src/Avalonia.Controls.TreeDataGrid/Models/TreeDataGrid/ColumnOptions.cs

```diff
@@ -45,5 +80,10 @@ public class ColumnOptions<TModel>
         /// Gets or sets a custom comparison for descending ordered columns.
         /// </summary>
         public Comparison<TModel?>? CompareDescending { get; set; }
+
+        /// <summary>
+        /// Gets or sets the gesture(s) that will cause a cell to enter edit mode.
+        /// </summary>
+        public EditGestures EditGestures { get; set; } = EditGestures.Default;
```

Comment:
I wonder if it makes sense adding EditGesture to every column type. 
Like, in CheckBox or ComboBox columns it won't make much sense.

Reply:
Well I think that in the WPF data grid, those column types do require entering edit mode (which IMO is undesirable default behavior). We might want to add a "use edit mode" option for those, in which case this would apply?